### PR TITLE
SIMD: i16x8.splat i32x4.splat i64x2.splat f32x4.splat f64x2.splat ins…

### PR DESCRIPTION
**Pull request by lizhengxing**
…truction implementation.

### CODE REVIEW

**Comment by lizhengxing** [comment]
@binji 
This PR implements the simd i16x8 / i32x4 / i64x2 / f32x4 / f64x2.splat instructions. PTAL, Thanks!

@arunetm FYI!

File: src/interp.cc [hunk 1]
```diff
@@ -2249,6 +2249,36 @@ Result Thread::Run(int num_instructions) {
         break;
       }
 
+      case Opcode::I16X8Splat: {
+        uint16_t dLane = static_cast<uint16_t>(Pop<uint32_t>());
```
Comment (binji) [inline src/interp.cc]: nit: `lane`

Comment (binji) [reply src/interp.cc]: nit: `static_cast` is not needed since `Pop<T>` returns type `T` -- here and below

Comment (lizhengxing) [reply src/interp.cc]: Done.

**Comment by lizhengxing** [comment]
@binji 
Update the PR according to feedbacks, PTAL, thanks!

@arunetm FYI!

**Pull request merged**